### ISSUE TITLE
No /proc/mounts on FreeBSD

### ISSUE THREAD

**Issue by bsdlme** [mucommander/mucommander#397]
On FreeBSD there's no /proc/mounts file, so muCommander cannot read the currently mounted partitions.
You can instead run `/sbin/mount -p` which produces the exact same output:

```
mount -p
zroot/ROOT/nomadbsd-1.3.1 /                     zfs     rw,noatime,nfsv4acls    0 0
devfs                   /dev                    devfs   rw              0 0
procfs                  /proc                   procfs  rw              0 0
tmpfs                   /tmp                    tmpfs   rw              0 0
zroot/var/tmp           /var/tmp                zfs     rw,nosuid,noatime,nfsv4acls     0 0
zroot/var/log           /var/log                zfs     rw,noexec,nosuid,noatime,nfsv4acls      0 0
zroot/usr/home          /usr/home               zfs     rw,noatime,nfsv4acls    0 0
zroot/var/crash         /var/crash              zfs     rw,noexec,nosuid,noatime,nfsv4acls      0 0
zroot/var/audit         /var/audit              zfs     rw,noexec,nosuid,noatime,nfsv4acls      0 0
zroot/usr/obj           /usr/obj                zfs     rw,noatime,nfsv4acls    0 0
zroot/usr/ports         /usr/ports              zfs     rw,nosuid,noatime,nfsv4acls     0 0
zroot/usr/src           /usr/src                zfs     rw,noexec,nosuid,noatime,nfsv4acls      0 0
zroot/var/mail          /var/mail               zfs     rw,noexec,nosuid,nfsv4acls      0 0
zroot/usr/home/ports    /usr/home/ports         zfs     rw,noatime,nfsv4acls    0 0
zroot/usr/ports/packages /usr/ports/packages    zfs     rw,noexec,nosuid,noatime,nfsv4acls      0 0
zroot/usr/ports/distfiles /usr/ports/distfiles  zfs     rw,noexec,nosuid,noatime,nfsv4acls      0 0
linprocfs               /compat/linux/proc      linprocfs       rw              0 0
linsysfs                /compat/linux/sys       linsysfs        rw              0 0
devfs                   /compat/linux/dev       devfs   rw              0 0
fdescfs                 /compat/linux/dev/fd    fdescfs rw              0 0
tmpfs                   /compat/linux/dev/shm   tmpfs   rw              0 0
fdescfs                 /dev/fd                 fdescfs rw              0 0
/usr/home               /compat/linux/home      nullfs  rw,noatime,nfsv4acls    0 0
/tmp                    /compat/linux/tmp       nullfs  rw              0 0
```

**Comment by ahadas** [comment]
hi @bsdlme , thanks for reporting this.
could you please check if it works properly with the [nightly build](https://github.com/mucommander/mucommander/releases/download/nightly/mucommander-0.9.6-snapshot-portable.zip)?

**Comment by bsdlme** [comment]
Yes, thank you for handling the issue so fast!

**Comment by ahadas** [comment]
> Yes, thank you for handling the issue so fast!

thanks for the confirmation!

**Comment by ahadas** [comment]
This is fixed in [v0.9.6](https://github.com/mucommander/mucommander/releases/tag/0.9.6-1)
